webapp/sources/rudder/rudder-core/src/main/scala/com/normation/rudder/facts/nodes/NodeFactRepository.scala

```diff
@@ -689,9 +695,7 @@ class CoreNodeFactRepository(
     internalSave(Right((nodeId, tag)))(cc, SelectFacts.none)
   }
 
-  override def save(
-      nodeFact: NodeFact
-  )(implicit cc: ChangeContext, attrs: SelectFacts = SelectFacts.all): IOResult[NodeFactChangeEventCC] = {
+  override def save(nodeFact: NodeFact)(implicit cc: ChangeContext, attrs: SelectFacts): IOResult[NodeFactChangeEventCC] = {
```

Comment:
no default value for implicit, else calling sites don't know they are not doing what they should.